package.json

```diff
@@ -59,6 +59,7 @@
     "debug": "^2.2.0",
     "levee": "^1.2.0",
     "lodash": "^4.2.0",
+    "qs": "^6.3.1",
```

Comment:
Do we need this dependency? Node has a native module for this.

https://nodejs.org/docs/latest-v7.x/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options